### ISSUE TITLE
Summer Treasure

### ISSUE THREAD

**Issue by FrogInSave** [ValveSoftware/Dota2-Gameplay#2370]
### Description

I bought Summer Treasure for the fragments and did not get a courier, although all the other skins have already been received. Got Wraith King again. Help.

**Comment by shaileshgarg** [comment]
Yeah, thats normal. You get the Common Immortal after you have 1 of all of the Commons, and this repeats. The Rare, Very Rare and Ultra Rare are chance based.
